extern/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+include( FetchContent )
+
+FetchContent_Declare(
+    googletest
+    GIT_REPOSITORY https://github.com/google/googletest.git
+    GIT_TAG        release-1.10.0
```

Comment:
I believe the tag used here should be v1.10.x to get bugfixes for 1.10.

Reply:
Changed as requested.